app/pom.xml

```diff
@@ -174,6 +174,7 @@
     <module>s2i</module>
     <module>meta</module>
     <module>ui-react</module>
+    <module>test</module>
```

Comment:
Not sure about this change, I think `test` and `../doc` module are included in most cases (i.e. when `non-staging` profile is active), and we don't want those to be included when staging (I guess, no idea what sets `stagingDescription`, could be Sonatype release plugin).

Reply:
Removing it then.